staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5596,6 +5596,13 @@ const (
 	TLSCertKey = "tls.crt"
 	// TLSPrivateKeyKey is the key for the private key field in a TLS secret.
 	TLSPrivateKeyKey = "tls.key"
+	// TLSCACertKey is the intermediate CA Certificate chain for client authentication in a TLS secret
+	TLSCACertKey = "ca.crt"
```

Comment:
This does not match the [Ingress documentation](https://kubernetes.github.io/ingress-nginx/examples/auth/client-certs/#creating-certificate-secrets)... ca.crt refers to a CA used to verify presented client certificates, not the intermediate CA chain for this certificate.

An intermediate chain for the certificate can be included in `tls.crt`.

Reply:
From the same documentation: `Note: The CA Certificate must contain the trusted certificate authority chain to verify client certificates.`

So, indeed, it is for the client cert verification. Though I did not state that it is for the server cert.